lego/apps/surveys/views.py

```diff
@@ -22,7 +25,27 @@ def get_serializer_class(self):
             return SurveyUpdateSerializer
         elif self.action in ['retrieve']:
             return SurveyReadDetailedSerializer
-        return SurveyReadSerializer
+
+        return super().get_serializer_class()
+
+    @list_route(methods=['GET'])
+    def templates(self, request):
+        queryset = Survey.objects.all().prefetch_related('questions').filter(
+            template_type__isnull=False
+        )
+        serializer = self.get_serializer(queryset, many=True)
+        return Response(serializer.data)
+
+    @list_route(
```

Comment:
should these possibly be in their own `TemplateViewSet` with list and retrieve? might be a bit cleaner. you can still get the URLs you're using by doing something like this: https://github.com/webkom/lego/blob/master/lego/api/v1.py#L139

Reply:
Yeah you're right, that would be cleaner. Will fix

Reply:
could also just put the template viewset directly on the API root, i.e. `/api/v1/survey-templates` instead of `/api/v1/surveys/templates` - up to you